README.md

```diff
@@ -8,25 +8,24 @@ The SpatialOS Game Development Kit (GDK) for Unity enables game developers to bu
 
 * the example game, which both tests and demonstrates the Feature Modules; for use as a starting point or resource for game development.
 
-For further high-level information about this, see the [summary of key features](docs/content/faqs/key-features.md).
-
 This is an alpha release of the GDK Core which is the foundation of the product. To find out more see the [GDK blogpost](https://improbable.io/games/blog/unity-gdk-our-first-steps).
 
 ---
-* [Setup Guide](docs/setup-and-installing.md)
-* [Documentation](docs/README.md)
+
+* [Setup Guide](https://docs.improbable.io/unity/alpha/setup-and-installing#set-up-your-machine)
+* [Documentation](https://docs.improbable.io/unity/alpha/)
 * [Development Roadmap](https://trello.com/b/29tMKyQC)
-* [FAQs](docs/content/faqs/faqs.md)
 * [Discord](https://discord.gg/SCZTCYm)
 * [Forums](https://forums.improbable.io/latest?tags=unity-gdk)
 * [Github Issues](https://github.com/spatialos/UnityGDK/issues)
-* [Known Issues](docs/known-issues.md)
+* [Known Issues](https://docs.improbable.io/unity/alpha/known-issues)
```

Comment:
No "GDK for Unity" reference in this Url?

Reply:
There is not!